compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp

```diff
@@ -345,24 +343,22 @@ struct QuantizeWeights final : public luci::CircleNodeMutableVisitor<bool>
       {
         auto circle_const = loco::must_cast<luci::CircleConst *>(circle_node);
 
-        // Find min/max on the fly
-        float min = std::numeric_limits<float>::max();
-        float max = std::numeric_limits<float>::min();
-        for (uint32_t i = 0; i < circle_const->size<loco::DataType::FLOAT32>(); i++)
+        // Find min/max per channel-wise
+        if (granularity == QuantizationGranularity::ChannelWise)
         {
-          auto data = circle_const->at<loco::DataType::FLOAT32>(i);
-          min = data < min ? data : min;
-          max = data > max ? data : max;
+          // will be implemented.
+        }
+        // Find min/max per layer-wise
+        else
+        {
+          // Quantize using recorded quantparam
+          auto quantparam = circle_node->quantparam();
```

Comment:
maybe it would be better to assert `quantparam != nullptr`.

Reply:
I applied it :-)